conf/gulp-tasks/library.gulpfile.js

```diff
@@ -19,6 +20,23 @@ const uglify = require('gulp-uglify-es').default;
 
 const NAMESPACE = 'ANSWERS';
 
+function getLibVersion () {
+  const hash = require('child_process')
+    .execSync('git rev-parse HEAD')
+    .toString().trim();
+
+  const version = require('child_process')
+    .execSync('git describe --match "v[0-9]*" --abbrev=0 HEAD --always')
```

Comment:
I tried playing around with this locally for fun, and this command hasn't been working for me :( not sure what I'm doing wrong
![image](https://user-images.githubusercontent.com/23005393/83579130-6caec000-a506-11ea-9902-7159ebf2d33f.png)

Reply:
I think my issue is not I'm not using an annotated git tag, do we need to support lightweight tags as well?

Reply:
This is surprising. I would expect all of our release tags to be annotated tags instead of lightweight tags but that is not the case. It seems the Github UI does not allow for annotated tags. (All of our dist tags are annotated tags). @bryanreed What do you suggest here?

https://stackoverflow.com/questions/11514075/what-is-the-difference-between-an-annotated-and-unannotated-tag#:~:text=Annotated%20tags%20are%20meant%20for,private%20or%20temporary%20object%20labels.&text=annotated%20tags%20can%20contain%20a,without%20making%20a%20release%20commit.

Reply:
Using lightweight tags.